tests/sentry/eventstream/kafka/test_consumer_strategy.py

```diff
@@ -0,0 +1,68 @@
+import time
+from datetime import datetime
+from unittest.mock import Mock, patch
+
+import pytest
+from arroyo.backends.kafka import KafkaPayload
+from arroyo.types import Message, Partition, Topic
+
+from sentry.eventstream.kafka.consumer_strategy import PostProcessForwarderStrategyFactory
+from sentry.utils import json
+
+
+def get_kafka_payload() -> KafkaPayload:
+    return KafkaPayload(
+        key=None,
+        value=json.dumps(
+            [
+                2,
+                "insert",
+                {
+                    "group_id": 43,
+                    "event_id": "fe0ee9a2bc3b415497bad68aaf70dc7f",
+                    "organization_id": 1,
+                    "project_id": 1,
+                    "primary_hash": "311ee66a5b8e697929804ceb1c456ffe",
+                },
+                {
+                    "is_new": False,
+                    "is_regression": None,
+                    "is_new_group_environment": False,
+                    "skip_consume": False,
+                },
+            ]
```

Comment:
Now that we have more use cases with different schemas (see performance issues) and that, with arroyo, we can test the consumer logic much more thoroughly do you have a plan to improve the test coverage of the forwarder overall ?

Reply:
Honestly not really. The forwarder does not concern itself with schemas at all and has close to no product logic. It just forwards a message of any shape. The underlying dispatch function was not touched at all.